README.md

```diff
@@ -4,99 +4,91 @@
 
 ### Question #1
 
-Describe the differences between a SQL and NoSQL DB, and when you might use each.
+ Describe the differences between a SQL and NoSQL DB, and when you might use each.
 
-```text
-Your answer...
 
-```
+SQL databases are primarily relational, table-based databases, whereas NoSQL databases are non-relational and primarily use key value pairs and groups. SQL databases have predefined schema for their relationships (as in rails), whereas NoSQL do not have separate pre-defined schema files.    
+ ```
 
-### Question #2
+ ### Question #2
+ What's wrong with this mongoose code and how might we fix it? (Hint: Assuming there is a document with a name of "Bob", why is results not an author model on the second line?)
 
-What's wrong with this mongoose code and how might we fix it?
-(Hint: Assuming there is a document with a name of "Bob", why is results not an author model on the second line?)
+ var results = AuthorModel.find({name: "Bob"});
+ console.log(results);
 
-```js
-var results = AuthorModel.find({name: "Bob"});
-console.log(results);
-```
 
-```js
-// Your answer...
-```
+// The code does not use the callback function required in the Mongoose DB. You could call it using function(results) as the second attribute to the find function.
 
-### Question #3
+ ### Question #3
 
 Convert the following ActiveRecord sequence to Mongoose:
 
-```rb
-@andy = Instructor.find_by(name: "Andy")
-@andy.wishlist_items.create(description: "Resin Laying Deer Figurine, Gold")
-```
+Instructor.findOne({name: "Andy"}, function(err, andy){
+  andy.wishlist_items.push({description: "Resin Laying Deer Figurine, Gold");
+  andy.save();
+};
+});
+ ```
 
-```js
-// Your answer...
-```
+ ### Question #4
+Convert the following create method in Mongoose to ActiveRecord.
+ }
+ ```
 
-### Question #4
+-```rb
+```@author = Author.create!(name: params[:name])
 
+ ```
+ ## Express
 Convert the following create method in Mongoose to ActiveRecord.
 
-```js
-  var authors = {
-  create: function(req, res){
-  var author = new AuthorModel({name: req.body.name})
-  author.save(function(err){
-    if (!err){
-      res.redirect("authors")
-    }
-  })
-  }  
-}
-```
-
-```rb
+ ### Question #5
+ How does module.exports help us with separation of concerns?
 
-```
-## Express
 
-### Question #5
+module.exports is the thing (sometimes an object) that is returned as the result of a require call.
```

Comment:
Kind of, it is useful to help us make our code more modular, and we are using it to have access to code in different script files on the server side.